addons/test_mail/security/ir.model.access.csv

```diff
@@ -29,3 +29,4 @@ access_mail_test_track_compute,mail.test.track.compute,model_mail_test_track_com
 access_mail_test_track_monetary,mail.test.track.monetary,model_mail_test_track_monetary,base.group_user,1,1,1,1
 access_mail_test_track_selection_portal,mail.test.track.selection.portal,model_mail_test_track_selection,base.group_portal,0,0,0,0
 access_mail_test_track_selection_user,mail.test.track.selection.user,model_mail_test_track_selection,base.group_user,1,1,1,1
+access_mail_test_track_monetary_no_company,mail.test.track.monetary.no_company,model_mail_test_track_monetary_no_company,base.group_user,1,1,1,1
```

Comment:
Could be moved right under ``access_mail_test_track_monetary`` definition in the file, just to have a clean ordering :)

Reply:
okay